packages/wl-freeze/wl-freeze.sh

```diff
@@ -54,13 +67,15 @@ freezer() {
             # Race condition: if these processes are exiting, these
             # freeze files may no longer exist. Ignore failures.
             echo 1 > "$last_scopedir/cgroup.freeze"  || true
+            echo $last_scopedir >> $scope_cache
         fi
 
         # Race condition: if these processes are exiting, these
         # freeze files may no longer exist. Ignore failures.
         systemctl kill --user --signal CONT \
                   "$(echo "$scope" | rev | cut -d'/' -f1 | rev)" || true
```

Comment:
We should actually delete this CONT bit, since we don't SIGSTOP anymore, either.